auth_service/auth_adapter/api/main.py

```diff
@@ -62,7 +66,20 @@ async def ext_auth(
     but the status will still be returned as OK so that all requests can pass.
     """
     access_token = get_bearer_token(authorization, x_authorization)
-    internal_token = exchange_token(access_token)
+    if access_token:
+        # check whether the external id is of interest and only pass it on in that case
+        pass_sub = (path == "users" and request.method == "POST") or (
+            path.startswith("users/") and "@" in path and request.method == "GET"
+        )
```

Comment:
PMO: maybe at one point we can specify this as config parameter, so that it can be decided on at deploy time.

Reply:
I have extracted this to a function for now.